xcube/webapi/context.py

```diff
@@ -293,7 +292,7 @@ def get_dataset_place_groups(self, ds_id: str, load_features=False) -> List[Dict
         return place_groups
 
     def get_dataset_place_group(self, ds_id: str, place_group_id: str, load_features=False) -> Dict:
-        place_groups = self.get_dataset_place_groups(ds_id, load_features=False)
+        place_groups = self.get_dataset_place_groups(ds_id, load_features=load_features)
```

Comment:
`load_features=False` was *not* intended, see following `for` loop, where `load_features` is used to only load the desired place.